examples/dataframe/app.py

```diff
@@ -93,12 +93,12 @@ def handle_edit():
     @render.text
     def detail():
         if (
-            input.grid_selected_rows() is not None
-            and len(input.grid_selected_rows()) > 0
+            grid.input_selected_rows() is not None
+            and len(grid.input_selected_rows()) > 0
         ):
             # "split", "records", "index", "columns", "values", "table"
 
-            return df().iloc[list(input.grid_selected_rows())]
+            return df().iloc[list(grid.input_selected_rows())]
```

Comment:
Please, remove this line as the top comment as the suggested code. 

(Could not suggest a full change in a single comment. ...weird